src/django/api/mail.py

```diff
@@ -9,38 +9,86 @@
 logger = logging.getLogger(__name__)
 
 
-def send_boundary_submitted_validator_email(request, updated_boundary):
-    subj_template = get_template('mail/boundary_submitted_validator_subject.txt')
-    body_template = get_template('mail/boundary_submitted_validator_body.txt')
+def send_boundary_submitted_validator_email(request, boundary):
+    (subject, body) = get_boundary_subject_and_body(
+        request,
+        boundary,
+        subj_template_name='mail/boundary_submitted_validator_subject.txt',
+        body_template_name='mail/boundary_submitted_validator_body.txt',
+    )
 
-    validators = updated_boundary.utility.state.users.filter(role=Roles.VALIDATOR)
+    validators = boundary.utility.state.users.filter(role=Roles.VALIDATOR)
     validator_emails = [user.email for user in validators.only('email')]
 
-    template_data = {
-        "city": updated_boundary.utility.address_city,
-        "pwsid": updated_boundary.utility.pwsid,
-        "details_link": "{}/submissions/{}/".format(
-            make_iow_url(request),
-            updated_boundary.id,
-        ),
-    }
+    for email in validator_emails:
+        safe_send_single_recipient_mail(subject, body, recipient=email)
+
+
+def send_boundary_submitted_contributor_email(request, boundary):
+    (subject, body) = get_boundary_subject_and_body(
+        request,
+        boundary,
+        subj_template_name='mail/boundary_submitted_contributor_subject.txt',
+        body_template_name='mail/boundary_submitted_contributor_body.txt',
+    )
+
+    recipient = boundary.latest_submission.submitted_by.email
+
+    safe_send_single_recipient_mail(subject, body, recipient=recipient)
```

Comment:
Instead of sending an email to just the person who submitted it, let's send an update email to every contributor that has access to this utility, similar to how we handle the  validators.

```suggestion
    contributors = boundary.utility.users.filter(role=Roles.CONTRIBUTOR).values_list(
        "email", flat=True
    )

    for recipient in contributors:
        safe_send_single_recipient_mail(subject, body, recipient=recipient)
```